ds-controller-board/device/device.go

```diff
@@ -20,13 +20,13 @@ type ControllerBoard interface {
 // NewControllerBoard is used to determine the ControllerBoard type
 // (i.e. Device or Virtual), and perform the necessary steps to initialize
 // a new ControllerBoard
-func NewControllerBoard(lc logger.LoggingClient, asyncCh chan<- *dsModels.AsyncValues, config *Config) (ControllerBoard, error) {
+func NewControllerBoard(lc logger.LoggingClient, asyncCh chan<- *dsModels.AsyncValues, config *ServiceConfig) (ControllerBoard, error) {
```

Comment:
Could pass in config.DriverConfig to shorten the lines below
```suggestion
func NewControllerBoard(lc logger.LoggingClient, asyncCh chan<- *dsModels.AsyncValues, config *DriverConfig) (ControllerBoard, error) {
```